man/systemd.network.xml

```diff
@@ -1261,6 +1261,14 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
             unset.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Type=</varname></term>
+          <listitem>
+            <para>Specifies Routing Policy Database (RPDB) rule type. Takes one of <literal>blackhole</literal>,
+            <literal>unreachable</literal> or <literal>prohibit</literal>. When unset, the kernel's default will be used.
```

Comment:
The default value is `FR_ACT_TO_TBL`. It may not be equal to the kernel's default.

So, I think we need to add `FR_ACT_TO_TBL` to the string table (how about "pass-to-table"?), and document the default value.

Reply:
Better we ignore this if it's different;

Reply:
I checked the kernel code and `FR_ACT_TO_TBL` is not the default value in kernel. But `ip rule add` set it by default.

So, at least, please drop the last sentence: `When unset, the kernel's default will be used.`